cypress/support/commands.js

```diff
@@ -104,8 +104,12 @@ Cypress.Commands.add('mockTextFilterResponse', () => {
 
 Cypress.Commands.add('toggleLearnPageTab', tabName => {
     cy.get('[data-test="tabs"]').within(() => {
-        cy.contains(tabName).click();
+        cy.contains(tabName).should('exist').click();
     });
+    cy.get(`[data-test="tab-${tabName}"]`).should(
+        'have.css',
+        'border-bottom-color'
+    );
```

Comment:
There is probably a better way we can check for if the tab is now active here before moving on (maybe a comment here would do the trick as well?)

Reply:
Yeah, commenting here should do the trick.